freqtrade/commands/arguments.py

```diff
@@ -467,3 +470,14 @@ def _build_subcommands(self) -> None:
 
         self._build_args(optionlist=ARGS_LOOKAHEAD_ANALYSIS,
                          parser=lookahead_analayis_cmd)
+
+        # Add recursive_analysis subcommand
+        recursive_analayis_cmd = subparsers.add_parser(
+            'recursive-analysis',
+            help="Check for potential look ahead bias.",
```

Comment:
I think we might need a better help string.
This is identical to lookahead_analysis - but the command does not do exactly that.